.github/workflows/config.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.6, 3.7]
```

Comment:
Maybe just 3.7 and 3.8? Also we'll need to update the readme which currently says requires Python 3.5 or later (can do later). 3.5 may still work but we might as well say we support the latest two versions there.

Reply:
100% but when I plugged 3.8 in some minor bugs cropped up so I'd like to take care of that in a separate PR if that's ok.

For now this PR gives us "as is" with travis + appveyor.

I'll update the README though :+1:

Reply:
a0bb6f3